content/post/091-dev-exp.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2020-05-05
+url: 2020-05-05-developer-experience
+title: Improving the IPFS Developer experience
+author: Hector Sanjuan
+tags: Developer, Experience
```

Comment:
what header image do you want to include?

Reply:
Can go with defaults or do I need to come up with something custom?